graphql/api/src/main/resources/stream-registry.graphql

```diff
@@ -33,24 +33,48 @@ type Tag {
 
 ############ Specification ############
 
+input SecurityInput {
+    role: String!
+    principals: [String!]!
```

Comment:
Wasn't this `PrincipalInput` before? I think `SecurityInput` and `Security` should be complementary.

```
input PrincipalInput {
    name: String
}

input SecurityInput {
    role: String
    principals: [PrincipalInput!]!
}

type Principal {
    name: String
}

type Security {
    role: String
    principals: [Principal!]!
}
```

or 

```
input SecurityInput {
    role: String
    principals: [String!]!
}

type Security {
    role: String
    principals: [String!]!
}
```

Reply:
Done